content/cybersecurity/concepts/ransomware/ransomware.md

```diff
@@ -0,0 +1,25 @@
+---
+Title: 'Ransomware'
+Description: 'Ransomware is a cyber-attack that blocks and prevents a user from accessing important files or personal data that is located on a device or computer.'
+Subjects:
+  - 'Cybersecurity'
+  - 'Computer Science'
+Tags:
+  - 'Cybersecurity'
+CatalogContent:
+  - 'introduction-to-cybersecurity'
+  - 'paths/fundamentals-of-cybersecurity'
+---
+
+**Ransomware** is a [cyber attack](https://www.codecademy.com/resources/docs/cybersecurity/cyber-attack) that blocks and prevents a user from accessing important files or personal data that is located on a device or computer.
```

Comment:
Add the fact that the bad actors will ask for money, usually cryptocurrency, in order to restore the compromised system.